apps/client/src/components/atoms/text/text.css

```diff
@@ -0,0 +1,35 @@
+.font-major-mono-display {
+  font-family: 'Major Mono Display', monospace;
+}
+
+.font-roboto-mono {
+  font-family: 'Roboto Mono', monospace;
+}
+
+.font-sora {
```

Comment:
Please use tailwind classes instead of css

Reply:
Removed 60053a515725709ad13b0ffa3347a696845ad0b8